packages/medusa/src/services/__tests__/discount.js

```diff
@@ -99,6 +107,8 @@ describe("DiscountService", () => {
       })
 
       expect(discountRepository.save).toHaveBeenCalledTimes(1)
+      expect(eventBusService.emit).toHaveBeenCalledTimes(1)
+      expect(eventBusService.emit.mock.calls[0][0]).toEqual(DiscountService.Events.CREATED)
```

Comment:
suggestion: you can use `expect(eventBusService.emit).toHaveBeenCalledWith`

Reply:
> todo: you can use `expect(eventBusService.emit).toHaveBeenCalledWith`

Done. Sorry, hadn't seen the comment.